frontend/build/server.mjs

```diff
@@ -6,7 +6,7 @@ const watchDirectories = ["src/*.{ts,tsx}"];
 
 const runServer = async () => {
   const ctx = await esbuild.context(buildOptions);
-  await ctx.watch();
```

Comment:
Need to figure this bit out. Possibly going to do it with a Prod/Dev env variable